connectors/Makefile

```diff
@@ -2,6 +2,7 @@ ROOT_DIR := ..
 include $(ROOT_DIR)/Makefile.env
 
 CONNECTORS := \
+	katalog \
```

Comment:
Conflicts with the PR by @revit13, please coordinate if this is something that you continue. 

I'm not a fan of this Makefile by the way, I prefer each connector to decide what targets it wants to have and be executed independently (e.g., I don't really want to copy terminate or any of the deployments targets to katalog, I don't want to deploy both egeria and katalog, etc.)

Reply:
I agree there are too many targets here. I propose to remove the following targets from `connectors/Makefile`:
- build
- clean
- run
- terminate

These are all targets that are going to be called when working on or testing one connector.

Reply:
Okay, but this makefile is currently broken so either do it in this PR or remove this change. If done in this PR then please also;
- remove charts target
- add empty deploy/undeploy targets to katalog to keep this makefile working

Reply:
I kept the charts targets as they are as they're not based on the CONNECTORS array but explicitly specify the three connectors. I'll leave them until we're sure we have everything we need in helm and then we can remove them.

Reply:
Sure. They will override the file in charts so no one should execute them.